workload/s3/s3api/object_operations_type1.yaml

```diff
@@ -0,0 +1,23 @@
+test_1:
+    TEST_ID: TEST-40039
+    object_size:
+        0K: 200
+        1K: 679
+        10Kb: 1884
+        100Kb: 1787
+        1M: 1820
+        10M: 1670
+        100M: 156
+        1G: 3
+        2G: 1
+    write_percentage: 100  # Percentage of data to fill the storage.
+    read_percentage: 100  # Percentage of data to be used to read from storage.
+    delete_percentage: 100  # Percentage of data to be deleted from storage.
+    cleanup_percentage: 0  # Percentage of data once reached then cleanup all data.
+    #  If None then it will be fetched from cluster(cortx specific.)
+    total_storage_size: None  # Total available storage from cluster or user to be used.
+    min_runtime: 30d  # Minimum execution durations to mark workload pass.
+    sessions_per_node: 20  # Number of sessions per node.
```

Comment:
Session per node can be changed to 2.

Reply:
This is customizable parameter, user can update same? 
May I know why it should be default 2?

Reply:
Please keep it configurable through jenkins. 
We are currently executing the test with 2 sessions per worker node